protos/flyteidl/admin/version.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package flyteidl.admin;
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin";
+
+// Represents a request structure to get version of flyteadmin.
+message Version {
```

Comment:
can we follow this structure - https://github.com/flyteorg/flytestdlib/blob/master/version/version.go#L14-L21
Have 
Build, Version, BuildTime